lib/auth/auth.go

```diff
@@ -5252,8 +5252,8 @@ func (a *Server) Ping(ctx context.Context) (proto.PingResponse, error) {
 	}
 	features := modules.GetModules().Features().ToProto()
 
-	if a.accessMonitoringEnabled {
-		features.IdentityGovernance = a.accessMonitoringEnabled
+	if features.IdentityGovernance {
+		features.AccessMonitoring = a.accessMonitoringEnabled
```

Comment:
Does access list need to be included anywhere here at all?

Reply:
so the `access monitoring` feature requires both a license `features.IdentityGovernance` and that it be enabled in the cluster config in the auth service section

`access list` only requires the license

did that help?

Reply:
@smallinsky i think i did this check wrong. i think it should be:

```suggestion
	if a.accessMonitoringEnabled {
		features.AccessMonitoring = a.accessMonitoringEnabled
```

because without the license, we wanted to:
```
Only the first "free" security report that's available to everyone (access anti-patterns).
```

which is what user sees now? and any future access monitoring features should be limited (which if i understood correctly, it's still wip)